azure/errors.go

```diff
@@ -0,0 +1,9 @@
+package azure
+
+const (
+	publishSettingsConfigurationError = "PublishSettingsFilePath is set. Consequently ManagementCertificatePath and SubscriptionId must not be set."
+
+	managementCertificateConfigurationError = "ManagementCertificatePath is set. Consequently SubscriptionId must also be set, and PublishSettingsFilePath must not be set."
```

Comment:
also errors must be named as `ErrXxx` or `errXxx` (see go stdlib for convention)